charts/dapr/charts/dapr_operator/templates/dapr_operator_deployment.yaml

```diff
@@ -22,6 +22,20 @@ spec:
     spec:
       containers:
       - name: dapr-operator
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8080
+          initialDelaySeconds: 3
```

Comment:
This doesn't need to be configurable, as the user has no control over how long it will take the Dapr control plane to come up, and thus the user doesn't need to change these values.